qutebrowser/config/config.py

```diff
@@ -637,14 +638,22 @@ class StyleSheetObserver(QObject):
 
     """Set the stylesheet on the given object and update it on changes.
 
+    Note that if update is turned on and the stylesheet accesses any config
+    option with a syntax different from 'conf.abc.def' (with surrounding
```

Comment:
Maybe it would be a bit clearer to use `' conf.abc.def '`  (with spaces)

Actually, are the spaces required in jinja? Why not just search for config values without the spaces?

Reply:
Not very sure about that, just to be more strict because it could as well access something named `myconf` (?). Perhaps I can change it to use `\b`.

After all it should not be too problematic because the stylesheets are controlled by us (and not user defined)